apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/CreateLoadJobOptions.cs

```diff
@@ -115,6 +115,18 @@ public sealed class CreateLoadJobOptions : JobCreationOptions
         /// </summary>
         public EncryptionConfiguration DestinationEncryptionConfiguration { get; set; }
 
+        /// <summary>
+        /// Allows the schema of the destination table to be updated as a side effect 
+        /// of the load job if a schema is autodetected or supplied in the job configuration.
+        /// Schema update options are supported in two cases:
+        /// when<see cref="WriteDisposition"/> is <see cref = "WriteDisposition.WriteAppend" />;
```

Comment:
nit: please be consistent with spacing inside `see cref`; it looks like all others have no spaces surrounding `=` and before `/>`

Reply:
Done